setup.py

```diff
@@ -95,27 +106,56 @@
 
     # HTTP/REST bridge (also pulls in TLS packages!)
     'treq>=15.1.0',               # MIT license
+
 ]
-if sys.platform != 'win32':
-    # setproctitle does not provide wheels (https://github.com/dvarrazzo/py-setproctitle/issues/47) => disable on Windows
-    install_requires.append('setproctitle>=1.1.9')  # BSD license
 
-if sys.platform == 'win32':
-    install_requires.append('pypiwin32>=219')       # PSF license
+setproctitle_version = "1.1.9"  # BSD license
+pypiwin32_version = "219"  # PSF license
+pyinotify_version = "0.9.6"  # MIT license
+wsaccel_version = "0.6.2"  # Apache 2.0
+ujson_version = "1.33"  # BSD license
```

Comment:
Why are the versions separate from the requirement lines? They weren't so in the original nor in my PRs and I don't see why they should be.